spec/ttml1.xml

```diff
@@ -5071,7 +5071,7 @@ to the after edge.
 If four <loc href="#style-value-length">&lt;length&gt;</loc> specifications are provided, then they apply to before, end,
 after, and start edges, respectively.</p>
 <note role="clarification">
-<p>Pecentage values are relative to the dimension of the region to which they apply. For example, if the before and after edges correspond to the top and bottom edges of the region then percentage values that apply to either of the two edges are relative to the height of the region.</p>
+<p>Pecentage values are relative to the dimension of the region to which they apply. For example, if the before and after edges correspond to the top and bottom edges of the region, as is the case for Latin text where <att>tts:writingMode</att> is equal to <code>"lrtb"</code>, then percentage values that apply to either of the two edges are relative to the height of the region. Conversly, if the before and after edges correspond to the right and left edges of the region, as is the case for Japanese text where <att>tts:writingMode</att> is equal to <code>"tbrl"</code>, then percentage values that apply to either of the two edges are relative to the width of the region.</p>
```

Comment:
I do not agree with this change, as it is now attempting to explain (and paraphrase) the relevant XSL-FO semantics of what is meant by writing mode.

Reply:
@skynavga The proposed modification merely adds a second example. Were you not happy with the original text, which had a single example?

Reply:
I already accepted the original text (of line 5074 above shown in red). I do not accept the newest change that further elaborates the example.

Reply:
@skynavga is there an error in the newer change with the further elaborated example? It's certainly easier to grasp, and since it is in a clarifying note, that seems like a good thing.

Reply:
It now leaves the impression that it applies only to latin and japanese text. There is no need to explain writing mode in this context, and it produces clutter. Too much text is a bad thing and this example has now crossed the line.

Reply:
OK in general over-explaining is unhelpful. I don't agree that it leaves the impression that it only applies to latin and japanese text though. Could you please propose something that elaborates better than what was before but does not cross the line @skynavga ?

Reply:
Thanks for the clarifying text, @palemieux. This will help the user to understand the issue.

I agree with @nigelmegitt that the text does not give the impression to be exclusive for Latin or Japanese text. To address @skynavga concern the following could be added:

> as is the case for Latin text **(or other left-to-right scripts)** ...as is the case for Japanese text **(or other right-to-left scripts)**

I do not think that the commit is "over-explaining" or that it adds "clutter". On the contrary: it provides the necessary examples for users and TTML experts. That this question needed to be discussed in the TTWG and that the issue was possibly overseen is an indication that also the later group could make good use of it.